# Gibbs energy across temperatures

**Instructor solution.** Run this notebook independently from top to bottom.

In [1]:
from pathlib import Path
import sys

# Support kernels started in the repository root or in instructor-notebooks.
ROOT = Path.cwd()
if not (ROOT / "data").is_dir():
    ROOT = ROOT.parent
if not (ROOT / "data").is_dir():
    raise FileNotFoundError("Open this notebook from the repository or instructor-notebooks folder.")
sys.path.insert(0, str(ROOT))

In [2]:
deltaH = 1.52  # kJ/mol
deltaS = 0.0051  # kJ/(mol K)
temperatures = [100, 150, 200, 250, 300, 350, 400]
deltaG_list = []
spontaneous_temp_list = []
for temperature in temperatures:
    deltaG = deltaH - temperature * deltaS
    deltaG_list.append(deltaG)
    if deltaG < 0:
        spontaneous_temp_list.append(temperature)
print("Delta G (kJ/mol):", deltaG_list)
print("Spontaneous temperatures (K):", spontaneous_temp_list)

Delta G (kJ/mol): [1.01, 0.755, 0.5, 0.24499999999999988, -0.010000000000000009, -0.2650000000000001, -0.52]
Spontaneous temperatures (K): [300, 350, 400]


ΔG values are 1.01, 0.755, 0.500, 0.245, −0.010, −0.265, and −0.520 kJ/mol. The reaction is spontaneous at 300, 350, and 400 K under this model. Since ΔS is positive, increasing T lowers ΔG. The crossover is ΔH/ΔS ≈ 298.04 K; ΔG = 0 is the equilibrium boundary.